setup.sh

```diff
@@ -25,22 +26,43 @@ drupal8ci_install() {
 	# Add development dependencies to run the CircleCI jobs.
 	#
 	# behat/mink-extension is pinned until https://github.com/Behat/MinkExtension/pull/311 gets fixed.
-	composer require --dev \
-		cweagans/composer-patches \
-		behat/mink-extension:v2.2 \
-		behat/mink-selenium2-driver:^1.3 \
-		bex/behat-screenshot \
-		drupal/coder:^8.2 \
-		drupal/drupal-extension:master-dev \
-		drush/drush:~8.1 \
+	all_dev_deps=(
+		cweagans/composer-patches
+		behat/mink-extension:v2.2
+		behat/mink-selenium2-driver:^1.3
+		bex/behat-screenshot
+		drupal/coder:^8.2
+		drupal/drupal-extension:master-dev
+		drush/drush:~8.1
```

Comment:
Except from this one, all the other dependencies are dev ones, right? Therefore, how about if we have the following?

```
composer require drush/drush:~8.1
composer require --dev [all the others except composer-patches
```

We could print a message at the end of the script where we suggest to review the changes, commit and push. Thoughts?